MailCore/Cache/AccountManager/AccountManager.swift

```diff
@@ -204,6 +206,7 @@ public final class AccountManager: RefreshTokenDelegate, ObservableObject {
     private func createAndSetCurrentAccount(token: ApiToken) async throws -> ApiToken {
         let apiFetcher = MailApiFetcher(token: token, delegate: self)
         let user = try await userProfileStore.updateUserProfile(with: apiFetcher)
+        try await myKSuiteStore.updateMyKSuite(with: apiFetcher, id: user.id)
```

Comment:
This shouldn't break user login
```suggestion
        try? await myKSuiteStore.updateMyKSuite(with: apiFetcher, id: user.id)
```